content/misc/misc-12/contents.lr

```diff
@@ -5,7 +5,10 @@ title: How can I share files anonymously through Tor?
 seo_slug: share-files-anonymously-through-tor
 ---
 description:
+For sharing files over Tor, [OnionShare](https://onionshare.org/) is a good option. 
+OnionShare is temporarily/stealthily made available as a Tor Onion Service when a web server is started.
```

Comment:
It feels a bit ambiguous as well since OnionShare itself is freely available—it's the files shared over it that are temporarily/stealthily available. I suggest it might be a good idea to copy the description from OnionShare's repo itself.